pims/image_sequence.py

```diff
@@ -332,3 +332,33 @@ def __repr__(self):
                                   dtype=self.pixel_type,
                                   C=self._sizeC,
                                   Z=self._sizeZ)
+
+
+def customize_image_sequence(imread_func, name=None):
+    """Class factory for ImageSequence with customized image reader.
+
```

Comment:
Right. Specifically, this works for readers where each file is a separate frame. Otherwise, one must right their own subclass of `FramesSequence`.